QST-1006/first_JS_website/script.js

```diff
@@ -0,0 +1,7 @@
+    const toggleButton = document.getElementsByClassName("toggle-button")[0];
+
+    toggleButton.addEventListener('click', () => {
+        for(let i of document.getElementsByClassName('navlink') ){
```

Comment:
Pls use inbuilt map method since we spoke about es6 today and maybe assign a variable for navlink , like in line number 1 and use that variable in loop , just to keep it clean.